### ISSUE TITLE
demo Rmd files need inline documentation

### ISSUE THREAD

**Issue by titaniumtroop** [titaniumtroop/aRlegislation#3]
The Rmd files in the demo directory were used to train SVM and neural network models; the dataset used is a topical representation of each lawmaker, both on a career basis and on a session basis. The scripts produce images that help tune the hyperparameters in the files. However, the scripts do not contain inline documentation, which is needed to show the tuning process and hyperparameters used.

For the SVM model trained on the career-basis dataset, which was the most successful model with an F1 score of 80%, 9 iterations (numbered 0-8) were ultimately performed. The initial and final settings appear in the document, but the interim tuning steps were not preserved in the code. The strategies for tuning changes were as follows:

1. Select train/test split #s 4, 6, 13, 15, 18, 19, 22, 24; type is not polynomial, nu 0.01 to 0.05; cost >= 0.9; gamma <= 0.00136

2. nu >= 0.015; cost <= 17; gamma >= 0.001

3. type is not radial; 0.025 < nu < 0.075; nu-classification + linear type; C-classification + linear, sigmoid types

4. nu <= 0.05255; type is not sigmoid; 3 <= cost <= 10

5. drop C-classification

6. 0.0275 <= nu <= 0.0325

7. 0.028 <= nu <= 0.0305; select train/test split #18; 225 <= weight ratio <= 250,000; cross > 1

8. 145 <= topics <= 185; weight ratio >= 1000

Note that this was the tuning strategy; for hyperparameters defined on a log scale, I used 10^k, where k was a digit to 0-2 decimal places, to get reasonably close to the selected hyperparameter figure.
